test/specs/commands/set.js

```diff
@@ -259,7 +261,7 @@ describe('set command', () => {
 						});
 					});
 					Given('a variable "name"', given.aVariable, () => {
-						Given('a value "my_custom_lisky"', given.aValue, () => {
+						Given('a value "my_custom_lisk_cli"', given.aValue, () => {
```

Comment:
`my_custom_lisk_cli ` rather than `my_custom_lisk_commander`?

Reply:
I thought about this. "Lisk Commander" is our branded Lisk CLI. If people are making their own they might want their own branding.